frontend/src/components/nodes/NodeNav.tsx

```diff
@@ -0,0 +1,108 @@
+import Link from "next/link";
+
+import React from "react";
+import { Row, Col, Spinner } from "react-bootstrap";
+
+import NodesApi from "../../libraries/explorer-wamp/nodes";
+
+interface State {
+  validators?: number;
+  others?: number;
+}
+
+export default class extends React.Component<State> {
+  constructor(props: any) {
+    super(props);
+    this.timer = null;
+  }
+
+  timer: ReturnType<typeof setTimeout> | null;
+  state: State = {};
+
+  componentDidMount() {
+    this.timer = setTimeout(this.regularFetchInfo, 0);
+  }
+
+  componentWillUnmount() {
+    const timer = this.timer;
+    this.timer = null;
+    if (timer !== null) {
+      clearTimeout(timer);
+    }
+  }
+
+  regularFetchInfo = async () => {
+    await this.fetchNodes();
+    if (this.timer !== null) {
+      this.timer = setTimeout(this.regularFetchInfo, 10000);
+    }
+  };
+
+  fetchNodes = async () => {
+    const [
+      validators,
+      others
+    ] = await new NodesApi().getTotalValidatorsAndOther();
+    this.setState({ validators, others });
+  };
+
+  render() {
+    const { validators, others } = this.state;
+    return (
+      <>
+        <Row>
+          <Link href="/nodes/[role]" as={`/nodes/validators`}>
+            <a style={{ textDecoration: "none", marginLeft: "15px" }}>
+              <Col className="node-selector pagination-total align-self-center">
+                <img
+                  src={"/static/images/icon-m-node-validator-status.svg"}
+                  style={{ width: "12px", marginRight: "10px" }}
+                />
+                {validators ? (
+                  `${validators}  VALIDATOR NODES`
+                ) : (
+                  <Spinner animation="border" variant="secondary" />
+                )}
+              </Col>
+            </a>
+          </Link>
+          <Link href="/nodes/[role]" as={`/nodes/non-validators`}>
+            <a style={{ textDecoration: "none", marginLeft: "15px" }}>
+              <Col className="node-selector pagination-total align-self-center">
```

Comment:
We should have an indication of which tab is currenlty active. See the provided design

Reply:
they all active, one is active validators, another one is active non validators.

Reply:
I change it a little bit since if there if border, it is easy to see where to click and also good for mobile. I will change border color when actived.